.github/workflows/dockerimage.yml

```diff
@@ -2,22 +2,40 @@ name: Docker Image CI
 
 on:
   release:
-    branches: [ master ]    
+    branches: [ master ]
 
 jobs:
-
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2.7.0
-      - uses: docker/build-push-action@3e7a4f6646880c6f63758d73ac32392d323eaf8f # v1.1.2
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            netflixoss/metaflow_metadata_service
+          tags: |
+            type=semver,pattern={{raw}}
+            type=sha
+            type=raw,value=latest
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
is buildx a required step here since we only target the default `linux/amd64` platform here?

Reply:
Yeah, I was also not 100% sure whether to include this step. 

It's not strictly required, but the [documentation](https://github.com/docker/build-push-action#usage) of the `build-push-action` recommends its usage. 

Happy to remove it. Would you prefer it removed or kept?

Reply:
cool. That's done now.